src/components/templates/customer-referrals-table/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { CustomerReferralRedemption } from "@medusajs/medusa"
+import { RouteComponentProps } from "@reach/router"
+import moment from "moment"
+import { stringDisplayPrice } from "../../../utils/prices"
+import StatusDot from "../../fundamentals/status-indicator"
+import Table from "../../molecules/table"
+
+type CustomerReferralsTableProps = {
+  referrals: CustomerReferralRedemption[]
+} & RouteComponentProps
+
+const CustomerReferralsTable: React.FC<CustomerReferralsTableProps> = ({
+  referrals,
+}) => {
+  const decideStatus = (rewarded) => {
```

Comment:
this could be written with less lines, `const decideStatus = (rewarded: boolean) => !rewarded ? <StatusDot variant="warning" title={"Pending"} /> : <StatusDot variant="success" title={"Rewarded"} />`